ABMobility/Grid.pde

```diff
@@ -31,11 +32,25 @@ public class Grid {
      
      table = loadTable("block/Cooper Hewitt Buildings - Building Blocks.csv", "header");
      for (TableRow row : table.rows()) {
-       buildings.add(new Building(gridMap.get(row.getInt("id")),row.getInt("id"),row.getInt("R"),row.getInt("O"),row.getInt("A")));
+      // initialize buildings from data
+      int id = row.getInt("id");
+      int loc = id;  // initial location is same as building id
+      int capacityR = row.getInt("R");
+      int capacityO = row.getInt("O");
+      int capacityA = row.getInt("A");
+      Building b = new Building(gridMap.get(loc), id, capacityR, capacityO, capacityA);
+      buildings.add(b);
      }
      
-     for (int i=0;i<=18;i++){
-       buildingsOnGrid.add(new Building(gridMap.get(i),i,0,0,0));
```

Comment:
At this line the buildings that were previously instantiated were not being used and instead buildings with dummy values were being added to the grid.
In order to dynamically choose amenity locations, we should be able to choose buildings on the grid that have greater capacity for amenities.  That information was being overwritten.

Reply:
ok I see , there was not R,0 and A